src/core/midiPorts.cpp

```diff
@@ -468,4 +468,28 @@ void midiReceive(const MidiMsg& mm, const std::string& recipient)
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+std::string shortName(const std::string& port) {
+
+	std::string o = port;
+
+	// Remove non-alphanumeric chars
+	o.erase(std::remove_if(o.begin(), o.end(), 
+	[] (auto const& c) -> bool {return !std::isalnum(c);}), o.end());
```

Comment:
You can trust type inference of lambda return type since C++14:
```suggestion
	     [] (auto const& c) { return !std::isalnum(c); }),
	o.end());
```
also tweak the indent since the lambda is for `remove_if` while the last `o.end` apply to `erase`.
But honestly I do not know how nice multi-line indent can be performed while using tabs...

Reply:
Tabs AND strict 80-char lines. ;) I stick to that due to my environment setup.

Reply:
Probably this can work only if everybody has the exact same behaviour with tabs...